subsys/bluetooth/services/fast_pair/fp_storage.c

```diff
@@ -246,7 +246,7 @@ int fp_storage_account_key_count(void)
 	return account_key_count;
 }
 
-int fp_storage_account_keys_get(uint8_t buf[][FP_CRYPTO_ACCOUNT_KEY_LEN], size_t *key_count)
+int fp_storage_account_keys_get(struct fp_account_key buf[], size_t *key_count)
```

Comment:
Since you don't use two dimensional array you can keep buf parameter just as `struct fp_account_key *buf`